config/300-apiserversource.yaml

```diff
@@ -29,6 +29,7 @@ spec:
     - knative
     - eventing
     - sources
+    - importer
```

Comment:
I don't have a strong preference either way, but the line above is `sources`, should this be `importers`?

Reply:
I vote to keep sources too for now.

Reply:
I agree, we should keep `sources` for at least a full release.

My question was focused on `importer` vs `importers`.